Clear/main.py

```diff
@@ -0,0 +1,335 @@
+
+@neighbor.group(cls=AliasedGroup, default_if_no_args=True, context_settings=CONTEXT_SETTINGS)
+def soft():
+    pass
+
+neighbor.add_command(soft)
+
+@soft.command(default=True)
+@click.argument('ipaddress', required=False)
+def default(ipaddress):
+    """Clear BGP neighbors soft configuration"""
+    if ipaddress is not None:
+        command = 'sudo vtysh -c "clear ip bgp {} soft "'.format(ipaddress)
+    else:
+        command = 'sudo vtysh -c "clear ip bgp * soft"'
+    run_command(command)
+
+# 'soft in' subcommand
+@soft.command('in')
+@click.argument('ipaddress', required=False)
+def soft_in(ipaddress):
+        if ipaddress is not None:
+                command = 'sudo vtysh -c "clear ip bgp {} soft in"'.format(ipaddress)
+        else:
+                command = 'sudo vtysh -c "clear ip bgp * soft in"'
+        run_command(command)
+
+# 'soft out' subcommand
+@soft.command('out')
+@click.argument('ipaddress', required=False)
+def soft_in(ipaddress):
+        if ipaddress is not None:
+                command = 'sudo vtysh -c "clear ip bgp {} soft out"'.format(ipaddress)
+        else:
+                command = 'sudo vtysh -c "clear ip bgp * soft out"'
+        run_command(command)
+
+# 'ipv6' group
+
+@cli.group()
+def ipv6():
+    """Clear IPv6 information"""
+    pass
+
+
+# 'bgp' command
+@click.group(cls=AliasedGroup,default_if_no_args=True)
+def bgp():
+    """Clear IPv6 BGP (Border Gateway Protocol) information"""
+    pass
+
+ipv6.add_command(bgp)
+
+
+# Default 'bgp' command (called if no subcommands or their aliases were passed)
+@bgp.command(default=True)
+def default():
+    """Clear all BGP peers"""
+    command = 'sudo vtysh -c "clear ipv6 bgp *"'
+    run_command(command)
+
+@bgp.group(cls=AliasedGroup, default_if_no_args=True, context_settings=CONTEXT_SETTINGS)
+def neighbor():
+    pass
+
+bgp.add_command(neighbor)
+
+@neighbor.command(default=True)
+@click.argument('ipaddress', required=False)
+def default(ipaddress):
+    """Clear BGP neighbors"""
+    if ipaddress is not None:
+	command = 'sudo vtysh -c "clear bgp ipv6 {} "'.format(ipaddress)
+    else:
+	command = 'sudo vtysh -c "clear bgp ipv6 *"'
+    run_command(command)
+
+# 'in' subcommand
+@neighbor.command('in')
+@click.argument('ipaddress', required=False)
+def neigh_in(ipaddress):
+	if ipaddress is not None:
+		command = 'sudo vtysh -c "clear bgp ipv6 {} in"'.format(ipaddress)
+	else:
+		command = 'sudo vtysh -c "clear bgp ipv6 * in"'
+        run_command(command)
+
+# 'out' subcommand
+@neighbor.command('out')
+@click.argument('ipaddress', required=False)
+def neigh_out(ipaddress):
+	if ipaddress is not None:
+	        command = 'sudo vtysh -c "clear bgp ipv6 {} out"'.format(ipaddress)
+	else:
+		command = 'sudo vtysh -c "clear ip bgp  ipv6 * out"'
+        run_command(command)
+
+@neighbor.group(cls=AliasedGroup, default_if_no_args=True, context_settings=CONTEXT_SETTINGS)
+def soft():
+    pass
+
+neighbor.add_command(soft)
```

Comment:
This line is unnecessary, as neighbor is added to the `neighbor` group via the decorator on line 275.